### DISCUSSION TITLE
Error in mimir query data

### DISCUSSION THREAD

**Discussion by Alexis-cmyk-a11y** [grafana/mimir#4138] [Help and support]
Error in mimir query data （which exceeds the maximum allowed staleness period of 1h0m0s (err-mimir-bucket-index-too-old)）

![image](https://user-images.githubusercontent.com/82816977/216217161-e754833d-2645-4255-a464-468d58d11ad4.png)

**Comment by charleskorn** [comment]
Hey @Alexis-cmyk-a11y, have you had a look at the runbook for this error (https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#err-mimir-bucket-index-too-old)?

**Reply by Alexis-cmyk-a11y** [reply]
How should it be repaired?

**Reply by charleskorn** [reply]
That will depend on what the cause of the problem is. Did the runbook help uncover any specific problems? (eg. an issue running compaction)
